google/cloud/pubsub/internal/tracing_batch_sink_test.cc

```diff
@@ -216,6 +217,33 @@ TEST(TracingBatchSink, PublishSpanHasAttributes) {
                                  TestTopic().FullName())))));
 }
 
+#if OPENTELEMETRY_VERSION_MAJOR >= 1
+#if OPENTELEMETRY_VERSION_MINOR >= 13
+TEST(TracingBatchSink, PublishSpanIsRoot) {
+  auto span_catcher = InstallSpanCatcher();
+  auto message_span = MakeSpan("test span");
+  auto mock = std::make_unique<pubsub_testing::MockBatchSink>();
+  EXPECT_CALL(*mock, AddMessage(_));
+  EXPECT_CALL(*mock, AsyncPublish)
+      .WillOnce([](google::pubsub::v1::PublishRequest const& request) {
+        EXPECT_THAT(request, IsProtoEqual(MakeRequest(1)));
+        return make_ready_future(make_status_or(MakeResponse(request)));
+      });
+  auto batch_sink = MakeTestBatchSink(std::move(mock));
+  auto initial_spans = {message_span};
+  AddMessages(initial_spans, batch_sink);
+
+  auto response = batch_sink->AsyncPublish(MakeRequest(1)).get();
+  EXPECT_THAT(response, IsOk());
+
+  auto spans = span_catcher->GetSpans();
+  EXPECT_THAT(spans,
+              Contains(AllOf(SpanNamed("test-topic publish"), SpanIsRoot())));
+  EXPECT_THAT(spans, Contains(AllOf(SpanNamed("test span"), SpanIsRoot())));
```

Comment:
I'm fine not including this expectation. If we are going to include it, I'd combine it with the one above.

Reply:
Ack, removed.